src/hammersbald.rs

```diff
@@ -179,12 +179,6 @@ impl ChainDB for Hammersbald {
     fn fetch_header(&self, id: &sha256d::Hash) -> Result<Option<StoredHeader>, Error> {
         Ok(self.db.get_hash_keyed::<StoredHeader>(id)?.map(|(_, header)| header))
     }
-
-    /// Shutdown db
-    fn shutdown(&mut self) {
```

Comment:
Shouldn't there be a `shutdown` function somewhere to stop the db? I think it should be moved into the Hammersbald structure method instead of just removing it.